src/contrib/cluster/Akka.DistributedData.Tests/LWWDictionarySpec.cs

```diff
@@ -131,12 +132,14 @@ public void LWWDictionary_must_be_able_to_work_with_deltas()
         /// Bug reproduction: https://github.com/akkadotnet/akka.net/issues/4400
         /// </summary>
         [Fact]
-        public void Bugfix_4400_LWWDictionary_Deltas_must_merge_other_LWWDictionary()
+        public async Task Bugfix_4400_LWWDictionary_Deltas_must_merge_other_LWWDictionary()
         {
             var m1 = LWWDictionary<string, string>.Empty
                 .SetItem(_node1, "a", "A")
                 .SetItem(_node1, "b", "B1");
 
+            await Task.Delay(200);
```

Comment:
There isn't anything `async` happening here though - why would this work?

Reply:
I just added a slight delay between setting the "b" item so the second set always have a higher timestamp compared to the first step.
I don't know how, but in net461, the second set can have an earlier timestamp if you leave out the delay.

Reply:
Ah I see